modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -346,3 +356,9 @@ public function renewLoans(AlephPatron $patron, array $ids) {
   }
 
 }
+
+/**
+ * Define exceptions for different error conditions inside the Aleph client.
+ */
+
+class AlephPatronInvalidPin extends Exception { }
```

Comment:
[One class for each file please](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#3-namespace-and-class-names).

Reply:
I agree. Just did it the way some of the others provider do. I'll change it. :)